reproman/interface/execute.py

```diff
@@ -190,25 +197,44 @@ def _prepare_remote(self):
         # TODO: might want to add also a "marker" so within the trace
         #       we could avoid retracing session establishing bits themselves
 
+        # Upload Docker shim
+        self.remote_docker_shim = op.join(remote_tracer_dir, "docker")
+        if not self.session.exists(self.remote_docker_shim):
+            self.session.put(self.local_docker_shim, self.remote_docker_shim)
```

Comment:
This has the potential to become stale if the shim is updated.  Given this is a small script, I think it'd be better to just put the file there (at least with local and ssh, `put` seems happy to overwrite the file).  To avoid the unnecessary copy we could even check whether the command is "docker".

Reply:
This has been fixed as suggested:

be05d09362196764e7bec940ceb2248f659afdda

Reply:
Thanks, @mjtravers.  From my POV this is good to go.